operations/service_web.go

```diff
@@ -222,7 +222,7 @@ func getAdminService(ctx context.Context, env evergreen.Environment, settings *e
 	localManagement.SetPrefix("/amboy/local/management")
 
 	apps = append(apps, localAbort, remoteAbort, groupAbort, localManagement)
-	if evergreen.EnableAmboyRemoteManagement {
+	if env.Settings().ServiceFlags.EnableAmboyRemoteManagement {
```

Comment:
use evergreen.ServiceFlags() to pull this from the DB rather than the cached settings object.